lib/SILGen/SILGenExpr.cpp

```diff
@@ -1693,7 +1693,7 @@ RValue RValueEmitter::visitMemberRefExpr(MemberRefExpr *E, SGFContext C) {
   // and releases.
   if (auto *N = E->getBase()->getType()->getNominalOrBoundGenericNominal())
     if (auto RV = NominalTypeMemberRefRValueEmitter(E, C, N).emit(SGF))
-      return RValue(std::move(RV).getValue());
+      return RValue(const_cast<RValue&&>(std::move(RV).getValue()));
```

Comment:
Why is this necessary? An rvalue reference-to-`const` has almost no reason to exist.